hal/src/main/native/athena/AnalogGyro.cpp

```diff
@@ -169,6 +169,8 @@ void HAL_CalibrateAnalogGyro(HAL_GyroHandle handle, int32_t* status) {
 
   HAL_InitAccumulator(gyro->handle, status);
   if (*status != 0) return;
+  wpi::outs() << "Callibrating analog gyro for " << kCalibrationSampleTime <<
```

Comment:
You forgot to include the header. Also, the formatting is wrong: https://dev.azure.com/wpilib/wpilib/_build/results?buildId=656&view=logs